File: app/src/main/java/com/scatl/uestcbbs/module/message/view/SystemMsgView.java
package com.scatl.uestcbbs.module.message.view;

import com.scatl.uestcbbs.entity.SystemMsgBean;

/**
 * author: sca_tl
 * description:
 * date: 2020/1/27 15:49
 */
public interface SystemMsgView {
    void onGetSystemMsgSuccess(SystemMsgBean systemMsgBean);
    void onGetSystemMsgError(String msg);
}

File: app/src/main/java/com/scatl/uestcbbs/module/message/view/SystemMsgView.kt
package com.scatl.uestcbbs.module.message.view

import com.scatl.uestcbbs.base.BaseView
import com.scatl.uestcbbs.entity.SystemMsgBean

/**
 * Created by sca_tl at 2023/2/20 10:23
 */
interface SystemMsgView: BaseView {
    fun onGetSystemMsgSuccess(systemMsgBean: SystemMsgBean)
    fun onGetSystemMsgError(msg: String?)
}